alf/examples/metadrive/muzero_metadrive_transformer_conf.py

```diff
@@ -0,0 +1,412 @@
+            bias_initializer=_get_rv_bias_initializer(),
+            kernel_initializer=torch.nn.init.zeros_),
+        *reshape_layer,
+    )
+
+    reward_net = [
+        partial(_summarize_grad, name='reward_grad'),
+        *_make_trunk(),
+        layers.FC(
+            dim,
+            num_quantiles,
+            weight_opt_args=dict(weight_decay=rv_weight_decay),
+            bias_opt_args=dict(weight_decay=rv_bias_decay),
+            bias_initializer=_get_rv_bias_initializer(),
+            kernel_initializer=torch.nn.init.zeros_),
+        *reshape_layer,
+    ]
+
+    reward_net = alf.nn.Sequential(*reward_net, input_tensor_spec=state_spec)
+
+    action_net = layers.Sequential(
+        partial(_summarize_grad, name='policy_grad'), *_make_trunk(),
+        BetaProjectionNetwork(dim, action_spec, min_concentration=1.0))
+
+    game_over_net = layers.Sequential(
+        *_make_trunk(),
+        layers.FC(
+            dim,
+            1,
+            kernel_initializer=torch.nn.init.zeros_,
+            bias_init_value=initial_game_over_bias),
+        layers.Reshape(()),
+    )
+
+    return alf.nn.Branch(
+        value_net,
+        reward_net,
+        action_net,
+        game_over_net,
+        input_tensor_spec=state_spec,
+    )
+
+
+lr_schedule = StepScheduler("percent", [(0.8, initial_lr),
+                                        (1.0, 0.05 * initial_lr)])
+
+alf.config(
+    "MCTSModel",
+    value_loss=rv_loss,
+    reward_loss=rv_loss,
+    repr_loss=losses.AsymmetricSimSiamLoss(
+        input_size=d_model,
+        proj_hidden_size=512,
+        pred_hidden_size=512,
+        output_size=1024),
+    predict_reward_sum=True,
+    policy_loss_weight=1.1,
+    value_loss_weight=0.5,
+    repr_prediction_loss_weight=20.0,
+    reward_loss_weight=2.0,
+    initial_loss_weight=1.0)
+
+alf.config(
+    "SimpleMCTSModel",
+    num_sampled_actions=16,
+    encoding_net_ctor=create_representation_net,
+    dynamics_net_ctor=create_ego_centric_dynamics,
+    prediction_net_ctor=create_prediction_net,
+    train_repr_prediction=True,
+    train_game_over_function=True,
+    initial_alpha=0.)
+
+alf.config(
+    "MCTSAlgorithm",
+    num_simulations=81,
+    num_parallel_sims=3,
+    root_dirichlet_alpha=0.3,
+    root_exploration_fraction=0.,
+    pb_c_init=1.5,
+    pb_c_base=19652,
+    is_two_player_game=False,
+    value_min_max_delta=0.01,
+    ucb_break_tie_eps=1e-6,
+    visit_softmax_temperature_fn=VisitSoftmaxTemperatureByProgress(
+        [(0.5, 1.0), (0.75, 0.5), (1, 0.25)]),
+    discount=discount,
+    act_with_exploration_policy=True,
+    learn_with_exploration_policy=True,
+    search_with_exploration_policy=True,
+    unexpanded_value_score='mean',
+    expand_all_children=False,
+    expand_all_root_children=False,
+    max_unroll_length=8,
+    learn_policy_temperature=1.0)
+
+alf.config(
+    "MuzeroRepresentationImpl",
+    model_ctor=SimpleMCTSModel,
+    num_unroll_steps=5,
+    td_steps=10,
```

Comment:
since reanalyze_ratio is 1.0, td_steps is not used. What really matter is reanalyze_td_steps

Reply:
`td_steps` is a required argument though. Added comment to note that it is not useful here, and also explicitly set `reanalyze_td_steps`.